Automattic/wp-calypso
# Making @automattic/components work in isolation

#### Changes proposed in this Pull Request

* renamed and moved `$sans`, `$sans-rtl`, `breakpoint` and `clear-fix` variables/mixins into `@automattic/components` with a prefix of `a8c-`
* updated `@automattic/components` to include any relevant styles defined from the Calypso reset/base styles (mostly just `font-family` related stuff)
* added stories for all of the `@automattic/components` to test them in isolation from the Calypso reset/base styles

**Future TODO**:
* Need to come up with a plan for handling `z-index` (used by a handful of the components) that scales across multiple apps

#### Testing instructions



* Check instances of fonts, breakpoints and clear fix across the app
* Check components in the Storybook visually look the same as the components in `/devdocs`
